pkg/apiserver/handlers_test.go

```diff
@@ -200,6 +200,35 @@ func TestTimeout(t *testing.T) {
 	}
 }
 
+func TestHasApiPrefix(t *testing.T) {
```

Comment:
> hasApiPrefix() does not modify the state of the requestAttributeGetter obj, so it could just be a func.

That's a good point, it probably doesn't belong on the interface and should just be a plain method.